data/presets/presets/playground/exercise.json

```diff
@@ -0,0 +1,23 @@
+{
+    "icon": "maki-playground",
+    "fields": [
+        "playground/theme",
+        "playground/min_age",
+        "playground/max_age",
+        "wheelchair"
+    ],
+    "moreFields": [
+        "name",
+        "playground/blind",
+        "playground/walking_disability",
+        "playground/sitting_disability",
+        "material"
+    ],
+    "geometry": [
+        "point"
+    ],
+    "tags": {
+        "playground": "exercise"
```

Comment:
I know this is somewhat different than `leisure=fitness_station` but perhaps not different enough to warrant a second tag…